build.py

```diff
@@ -67,10 +67,19 @@ def make_readable(html):
   # Beautiful soup breaks textarea formatting
   # since it adds extra whitespace. If you use "pre"
   # tags, you should be warry of the same problem
-  return re.sub(r'[\ \n]{2,}</textarea>',
+    
+  #The if statement is needed to maintain support for both python2 and python3	
+  if(type(prettyHTML)==str):
+    #For python2 the type is String
+    return re.sub(r'[\ \n]{2,}</textarea>',
                "</textarea>",
                prettyHTML)
-
+  else:
+    #For python3 the type is bytes
+    return re.sub(r'[\ \n]{2,}</textarea>',
+               "</textarea>",
+               prettyHTML.decode('utf-8'))
+     
```

Comment:
Having a single return statement would make this more [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).  One way to refactor with that aim would be conditionally decoding `prettyHTML` before it is used in the return statement.